package.json

```diff
@@ -70,6 +73,7 @@
     "rimraf": "^3.0.2",
     "semantic-release": "^17.4.7",
     "ts-jest": "^28.0.3",
+    "tsd-lite": "^0.5.4",
```

Comment:
from what i can tell `jest-runner-tsd` has a direct dependency to `tsd-lite`, which means `tsd-lite` should not have to be defined here at all, but `@tsd/typescript` would still be needed

https://github.com/jest-community/jest-runner-tsd/blob/ea6e070c9a37e7b402d47f23f3f6fc59bad5502f/package.json#L36

https://github.com/jest-community/jest-runner-tsd/blob/ea6e070c9a37e7b402d47f23f3f6fc59bad5502f/package.json#L56-L57

Reply:
That’s correct. This is just my habit to add dependencies to `package.json` explicitly, if something is imported from them.